app/src/main/java/com/khinthirisoe/bakingapp/ui/steps/StepsActivity.kt

```diff
@@ -0,0 +1,58 @@
+package com.khinthirisoe.bakingapp.ui.steps
+
+import android.content.Context
+import android.content.Intent
+import android.os.Build
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.khinthirisoe.bakingapp.R
+import com.khinthirisoe.bakingapp.data.model.Step
+import kotlinx.android.synthetic.main.activity_steps.*
+
+class StepsActivity : AppCompatActivity(), StepsContract.View {
+
+    companion object {
+        const val EXTRA_STEP = "extra_step"
+
+        fun createIntent(context: Context, step: Step) : Intent {
+            return Intent(context, StepsActivity::class.java).putExtra(StepsActivity.EXTRA_STEP, step)
+        }
+
+    }
+
+    private lateinit var presenter: StepsContract.Presenter
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_steps)
+
+        initView()
+    }
+
+    override fun onDestroy() {
+        super.onDestroy()
+        presenter.deactivate()
+    }
+
+    private fun initView() {
+        presenter = StepsPresenter(this)
```

Comment:
@KhinThiriSoe init the presenter in onCreate instead of `initView`  because the name initView means initiating views and the presenter is not a view